reactor-netty-http/src/main/java/reactor/netty/http/server/HttpTrafficHandler.java

```diff
@@ -506,8 +508,18 @@ protected void channelIdle(ChannelHandlerContext ctx, IdleStateEvent evt) {
 
 		static void addIdleTimeoutHandler(ChannelPipeline pipeline, @Nullable Duration idleTimeout) {
 			if (idleTimeout != null) {
-				String baseName = pipeline.get(NettyPipeline.HttpCodec) != null
-						? NettyPipeline.HttpCodec : NettyPipeline.H2CUpgradeHandler;
+				String baseName = null;
+				if (pipeline.get(NettyPipeline.HttpCodec) != null) {
+					baseName = NettyPipeline.HttpCodec;
+				} else if (pipeline.get(NettyPipeline.H2CUpgradeHandler) != null) {
```

Comment:
[checkstyle]: `else` is expected on new row

```suggestion
				}
				else if (pipeline.get(NettyPipeline.H2CUpgradeHandler) != null) {
```